### ISSUE TITLE
acc keeps 1?

### ISSUE THREAD

**Issue by FUtureLee759** [keras-team/keras#13131]
![myplot](https://user-images.githubusercontent.com/49342876/61613264-28edf780-ac61-11e9-95c1-e711f361dcfd.png)
you can see from this plot. The training acc and val_acc after first epochs keep 1. And corresponding loss are also very low. No matter how I change the parameters like epochs, batch, layers...It won't work... 

My work is about classification and I have in totally 3 classes, each has about 1800 samples. 

Can someone help me ?

**Comment by gowtham-kp** [comment]
For expedited response, @FUtureLee759 can you please provide me a code snippet to reproduce the issue. Thanks!

**Comment by FUtureLee759** [comment]
@gowtham-kp 
dataset1 = dataset100.reshape((1737, 14*11))
dataset2 = dataset175.reshape((2000, 14*11))
dataset3 = dataset250.reshape((1857, 14*11))

label1 = np.zeros(dataset1.shape[0], )
label2 = np.ones(dataset2.shape[0], )
label3 = 2 * np.ones(dataset3.shape[0], )


data = np.vstack((dataset1, dataset2, dataset3))
label = np.hstack((label1, label2, label3))
all = np.zeros((5594, 155))
all[:,:154] = data
all[:, 154] = label

np.random.shuffle(all)

x_train = all[:4000, :154]
x_test = all[4000:, :154]
y_train = all[:4000, 154]
y_test = all[4000:, 154]

x_val = x_train[:1000]
partial_x_train = x_train[1000:]
y_val = y_train[:1000]
partial_y_train = y_train[1000:]

y_test = to_categorical(y_test)
y_val = to_categorical(y_val)
partial_y_train = to_categorical(partial_y_train)



model = models.Sequential()
model.add(layers.Dense(64, activation="relu", input_shape=(11*14,)))
model.add(layers.Dense(64, activation="relu"))
model.add(layers.Dense(3, activation="softmax"))

model.compile(optimizer=optimizers.RMSprop(lr=0.001),
              loss='categorical_crossentropy',
              metrics=['accuracy'])

history = model.fit(partial_x_train,
                    partial_y_train,
                    epochs=20,
                    batch_size=32,
                    validation_data=(x_val, y_val))

**Comment by gowtham-kp** [comment]
@FUtureLee759 Let me know if these datasets dataset100, dataset175, dataset250 are publicly available or else can you please direct me to these datasets. Also can you please lower your learning rate and let me know if that helps in resolving the issue. Thanks!

**Comment by gowtham-kp** [comment]
Closing this issue due to the lack of recent activity. Please add additional  comments and we can open the issue again. Thanks!
